.github/workflows/ci.yml

```diff
@@ -4,27 +4,12 @@ on: [push, pull_request]
 
 jobs:
   build-macos:
-    runs-on: macOS-11
+    runs-on: macOS-14
     steps:
     # Checks-out the repo. More at: https://github.com/actions/checkout
     - uses: actions/checkout@v2
-    - name: Run on Xcode 11
-      run: sudo xcode-select -switch /Applications/Xcode_11.7.app
-    - name: Generate linux tests
-      # Makes sure the linux tests were generated properly (can only be run on macOS)
-      run: swift test --generate-linuxmain && git update-index --refresh && git diff-index --quiet HEAD --
```

Comment:
This may no longer exist:

```
error: Unknown option '--generate-linuxmain'
```

Not sure why we would need this since we also have the `build-linux` job.

Reply:
I don't know that much bout what this was doing. [here's what my linux tests tend to look like](https://github.com/dfed/SafeDI/blob/b50ff74f4543291934aff17122e0389cc3441ee2/.github/workflows/ci.yml#L102-L110)

Reply:
If you look at the logs in the `build-linux` job, it does run the tests. So, that part seems fine as-is.